doc/users/interactive_guide.rst

```diff
@@ -0,0 +1,86 @@
+.. _plotting-guide-interactive:
+
+*******************
+Interactive Figures
+*******************
+
+One of the most powerful uses of matplotlib is interactive
+figures.  At the most basic matplotlib has the ability to zoom and pan
+a figure to inspect your data, however there is also a full mouse and
+keyboard event handling system to enable building sophisticated interactive
+graphs.
+
+This page is meant to be a rapid introduction to the relevant details of
+integrating the matplotlib with a GUI event loop.
```

Comment:
Maybe add the reference: "For further details see: http://www.amazon.com/Interactive-Applications-using-Matplotlib-Benjamin/dp/1783988843?"